### PR TITLE
Testing with oktana card

### PR DESCRIPTION
**Pull request by SebasBL92**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: force-app/main/default/aura/oktanaCard/oktanaCard.cmp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    </p>
+                    <h3 class="slds-welcome-mat__tile-title">
+                      {!v.name}
+                    </h3>
+
+              </lightning:layoutItem>
+              <lightning:layoutItem padding="horizontal-small"  size="2" alignmentBump="left">
+
+                <div style="position:relative; transform: scale(0.8); top: 10%; left: calc(100% - 7vh);">
+
```
Comment (SebasBL92) [inline force-app/main/default/aura/oktanaCard/oktanaCard.cmp]: Testing...

File: force-app/main/default/aura/oktanaCard/oktanaCard.cmp [hunk 2]
```diff
@@ -14,7 +14,7 @@
   <aura:attribute name="titleSize" type="String" default="2" access="global"/>
 
   <aura:attribute name="dropableBody" type="String" default="" access="global"/>
-  <aura:attribute name="actions" type="Aura.Component[]" default="" access="global"/>
+  <aura:attribute name="actions" type="Aura.Component[]" default="" access="global"/>
```
Comment (SebastianVeraB) [inline force-app/main/default/aura/oktanaCard/oktanaCard.cmp:16]: Testing review

**Review by SebastianVeraB — CHANGES_REQUESTED** [review]

File: force-app/main/default/aura/oktanaCard/oktanaCard.cmp (hunk 2 above)
Comment (SebasBL92) [reply force-app/main/default/aura/oktanaCard/oktanaCard.cmp:16]: will fixing

**Review by SebastianVeraB — APPROVED** [review]
approved

**Pull request merged**